src/Symfony/Contracts/Cache/README.md

```diff
@@ -0,0 +1,9 @@
+Symfony Contracts
+=================
+
+A set of abstractions extracted out of the Symfony components.
```

Comment:
Here it should describe cache.

Reply:
I agree. I'd prefer doing it in a separate PR (could be done by someone else  :) )